src/components/Users/UsersContainer.js

```diff
@@ -81,11 +81,12 @@ UsersContainer.propTypes = {
   unfollow: PropTypes.func,
 };
 
-export default connect(
+
+export default withAuthRedirect(connect(
```

Comment:
Why is here used a different approach, without creating a separate variable?

Reply:
This is an example of multidimensional turnover. In the next lesson, I will get rid of this by making a compose